.cbmc-batch/stubs/aws_array_list_ensure_capacity_override.c

```diff
@@ -0,0 +1,22 @@
+#include <aws/common/array_list.h>
```

Comment:
1) Missing license header
2) Could you comment on what exactly is different in this implementation over the original one, and maybe also why we need the stub?

Reply:
The stub was used to improve verification time, but as it seems, verification time is reasonable with the standard implementation too, so I will remove it.